src/app/components/pages/UserProfile.jsx

```diff
@@ -176,8 +177,9 @@ export default class UserProfile extends React.Component {
             tab_content = (
                 <NotificationsList
                     username={accountname}
-                    notifications={notifications}
+                    notifications={notifications && notifications.toJS()}
```

Comment:
Most of our components seem to rely on immutablejs; it's a fairly consistent pattern. What's the reason for the switch here, and should we consider moving back to vanilla js as we move forward?

Reply:
Yeah, I wasn't too sure, happy to change this back. I guess vanilla JS is just easier to work with at the component level. I think I read somewhere it's best to keep immutable to the state layer, for example calling toJS() on each component render has performance ramifications, and React's props diffing might not detect changes to immutable objects... But again, happy to change this back if i am mistaken.